Fix the password urls and forms

PierrePaul opened the pull request:
Overwrites the url name to make sure the proper form is loaded and the proper url is used.

https://trello.com/c/pI9pKUN8/304-make-sure-that-the-forms-inherit-properly

pcraig3 commented on profiles/urls.py:
```diff
@@ -88,6 +88,7 @@
 )
 # Django.contrib.auth urls have underscore in them, let's change that for dashes
 urlpatterns += [
+    path("", include("django.contrib.auth.urls")),
```
Doesn't going first mean that it has precedence?

PierrePaul replied:
It does, but then we overwrite the route named 'password_reset' and make it point to our form instead.

pcraig3 commented:
The "new password" form having the colons. We want to get rid of the 'autocomplete' attributes that are in there.

![image](https://user-images.githubusercontent.com/2454380/91219706-11e1ca80-e6e9-11ea-9e26-a89cf0570add.png)

PierrePaul commented:
> The "new password" form having the colons. We want to get rid of the 'autocomplete' attributes that are in there, and inheriting from the base form removes that for us.

Since are overwriting all endpoints of django.contrib.auth.urls, I removed the include. I added a new Form that inherits HealthcareBaseForm to remove the `:` in the Confirm Reset Password view.
